packages/dashboard/src/components/contextMenu/menu.css

```diff
@@ -4,9 +4,7 @@
   position: absolute;
 }
 
-.iot-context-menu {
-  min-width: var(--size-context-menu-min-width);
-  border-radius: var(--radius-context-menu);
-  background-color: var(--colors-dark-grey);
-  z-index: var(--stack-order-context-menu);
+.iot-context-list-container {
+  padding: 0;
+  margin: -2px;
```

Comment:
what is this for? something around the border I'm guessing?

Reply:
yes.. else the on hover border will move the outer border and the UX looks shifty

Reply:
Does it match the same values as - `border: `${spaceScaledXxxs} solid ${colorBorderControlDefault}`? if so, can we make it match the same variable? Probably would have to move it into the style tag to do so. I don't think we'd catch that shifting behavior in any test so I could see us refactoring and causing that bug

Reply:
updated